mysql-test/suite/rpl/t/percona_bug1441259.test

```diff
@@ -0,0 +1,32 @@
+#
+# Bug lp:1441259 "super_read_only mode breaks replication with some queries"
+#
+--source include/have_binlog_format_statement.inc
+--source include/master-slave.inc
+
+# creating a few tables and fill them with some data
+connection master;
+CREATE TABLE t1 (
+  id INT NOT NULL PRIMARY KEY,
+  val INT NOT NULL
+);
+CREATE TABLE t2 (
+  id INT NOT NULL PRIMARY KEY,
+  val INT NOT NULL
+);
+INSERT INTO t1 VALUES (1, 1);
+INSERT INTO t2 VALUES (1, 2);
+sync_slave_with_master;
```

Comment:
--source include/sync_slave_sql_with_master.inc (sic?) is the recommended method these days